scipy/stats/_axis_nan_policy.py

```diff
@@ -362,6 +393,12 @@ def axis_nan_policy_wrapper(*args, _no_deco=False, **kwds):
             # Consolidate other positional and keyword args into `kwds`
             kwds.update(d_args)
 
+            keepdims = kwds.get("keepdims")
+            if keepdims is not None:
+                # remove keepdims to avoid passing it to the underlying
+                # hypotest function
+                kwds.pop("keepdims")
+
```

Comment:
```suggestion
```

Reply:
> Doesn't this work?

Oh yes, forgot `pop` takes another input as default return. I have changed it this way and moved it with other pops.